homeassistant/components/threshold/config_flow.py

```diff
@@ -76,4 +76,4 @@ class ConfigFlowHandler(SchemaConfigFlowHandler, domain=DOMAIN):
 
     def async_config_entry_title(self, options: Mapping[str, Any]) -> str:
         """Return config entry title."""
-        return options[CONF_NAME]
+        return cast(str, options[CONF_NAME])
```

Comment:
This cast shouldn't be needed?

Reply:
Mypy complains otherwise: `error: Returning Any from function declared to return "str"  [no-any-return]`

Reply:
Right, so instead of casting, assign it first.

```suggestion
        name: str = options[CONF_NAME]
        return name
```